syntax = "proto3";

message SkipperCommand {
	int32 sailPosition = 1;
	sint32 rudderPosition = 2;
}

message Mode {
	int32 mode = 1;
}
